lib/linkerd2-request-instrument/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "linkerd2-request-instrument"
```

Comment:
fwiw, there is already a crate that implements this behavior: https://github.com/tokio-rs/tracing/blob/master/tracing-tower/src/request_span.rs

it's in the nursery still but could be stabilized (basically requires better docs/examples)

Reply:
@hawkw i backed that out in e970e0d -- it only covered the response future and not the discover/resolution stream; and it also misses any problem the buffer task might have, so i've restored it to where it was.

Reply:
ah, okay. we should probably work on getting the `tracing-tower` layers in the stack separately, but that would be good to do...